src/funjson.c

```diff
@@ -64,6 +65,27 @@ json_escape_string(char *input)
   return buff;
 }
 
+/** Unescape a string for use as a JSON string. Returns a calloced buffer under 'json.string.partialbuf'. **/
```

Comment:
Needs a proper comment header documenting args and return values, and a more detailed/accurate description of what it's doing would be good - "unescape" isn't really descriptive of what it's doing

Reply:
New comment header description added. Let me know if the description works as is.